itemhandles.go

```diff
@@ -1 +1,59 @@
 package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+//ItemsRes does all this
+type ItemsRes struct {
+	Items []Item `json:"items"`
+}
+
+//GetItems is an http handler for finding an array of items and storing them in the items array
+func (corkboard *Corkboard) GetItems(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	items, _ := corkboard.findItems()
+	if items == nil {
+		log.Println("Items not found")
+	}
+
+	log.Println(items)
+	JSONobject, err := json.Marshal(items)
+	if err != nil {
+		log.Println("could not marshal items")
+	}
+	w.Write(JSONobject)
```

Comment:
This error is unchecked. w.Write() returns (CAS, err) so if you don't check the error the linter will yell at you.